### PR TITLE
Migration tool fixing merge conflicts 

### PR DESCRIPTION
**Pull request by rowasc**
This pull request makes the following changes:
-

Test checklist:
- [ ]

- [ ] I certify that I ran my checklist

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

File: app/Jobs/ExportPostsBatchJob.php [hunk 1]
```diff
@@ -6,8 +6,8 @@
 use Ushahidi\Factory\UsecaseFactory;
 use Ushahidi\Core\Entity\ExportJob;
 use Ushahidi\Core\Entity\ExportJobRepository;
-use Ushahidi\App\Multisite\MultisiteAwareJob;
 use Ushahidi\Core\Usecase\Post\Export;
+use Ushahidi\App\Multisite\MultisiteAwareJob;
 
 class ExportPostsBatchJob extends Job
```
Comment (rowasc) [inline app/Jobs/ExportPostsBatchJob.php]: this looks wrong  . Should have a 
-     use MultisiteAwareJob;

**Comment by tuxpiper** [comment]
mmm.. wait @rowasc, wouldn't we be actually looking into pushing the last "develop" onto the "migration-tool" branch instead?

My intention was to refresh the old branch and keep working on it.

**Comment by rowasc** [comment]
@tuxpiper yes

**Comment by rowasc** [comment]
@tuxpiper what I was doing was the same you tried to to originally (obviously branch should be renamed), but I avoided changing migration-tool directly because it was useful as a point of reference. I see you  have an up to date branch now, so I'll go ahead and close this one

**Pull request closed without merging**